src/tokens/transformation/skia/templates/colors.h.template

```diff
@@ -16,3 +16,27 @@ namespace leo::dark {
   return `constexpr SkColor ${prop.name} = SkColorSetRGB(${prop.value})`;
 }).map(s => `${s};`).join('\n') %>
 } // namespace leo::dark
+
+namespace leo {
+
+enum Theme {
+  kLight,
+  kDark
+};
+
+enum Color {
+<%= groupedTokens.light.allTokens.map(prop => `  ${prop.name}`).join(',\n') %>
+};
+
+SkColor GetColor(Color color, Theme theme) {
```

Comment:
I think it'd be good to migrate this into color provider facilities. Then, Given that, we could get colors from views like so,

```
void OnThemeChnaged() {
  GetColorProvider()->GetColor(kLeoFooColor);
}

```

But this would be available with changes from Brave-core

Reply:
Making this function `constexpr function` could be better.

Reply:
C++ can do some mad things :laughing: That's quite cool